mantidproject/mantid
# MergeRuns with time indexes

MergeRuns will now support workspaces with detector scans. There are two ways in which it can work:
1. DetectorInfo contains the same information - the spectra are summed.
1. DetectorInfo contains different, non-overlapping scan intervals - the spectra are appended to the first workspace.

**To test:**

Code review.

Test with the files attached. Load the two files and merge them, then check the merged workspace has the first workspace spectra (just counts in this case).

Try loading the same file into two different workspaces and then merge. Each count should be doubled.

[d2b_data.zip](https://github.com/mantidproject/mantid/files/1230676/d2b_data.zip)

Fixes #19927.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
